pallets/settlement/src/benchmarking.rs

```diff
@@ -147,14 +153,23 @@ fn generate_portfolio<T: Trait>(
     salt: u32,
     did: Option<IdentityId>,
 ) -> PortfolioId {
-    let pusedo_random_no = variable + salt;
-    let portfolio_no = (pusedo_random_no as u64).into();
+    let pseudo_random_no = variable + salt;
+    let portfolio_no = (pseudo_random_no as u64).into();
```

Comment:
I don't think we should be doing all of this. Instead, just call the create portfolio function of the portfolio module, please.